opentracing-api/src/main/java/io/opentracing/SpanScheduler.java

```diff
@@ -0,0 +1,87 @@
+package io.opentracing;
+
+/**
+ * Scheduler allows an existing (possibly thread-local-aware) execution context provider to also manage an
+ * actively-scheduled OpenTracing Span.
+ * <p>
+ * In any execution context (or any thread, etc), there is at most one "active" Span primarily responsible for the
+ * work accomplished by the surrounding application code. That active Span may be accessed via the
+ * {@link Scheduler#active()} method. If the application needs to defer work that should be part of the same
+ * Span, the Scheduler provides a {@link Scheduler#captureActive()} method that returns a
+ * {@link Continuation}; this activation context may be used to re-activate and deactivate the
+ * captured Span in that other asynchronous executor and/or thread.
+ */
+public interface Scheduler {
```

Comment:
I also think `Scheduler` is quite confusing. It reminds me of `java.util.concurrent.ScheduledExecutorService` and `org.springframework.scheduling.annotation.Scheduled` which is quite misleading. I'd be okay with `SpanManager` or  `TraceContext` (used in `jaeger-context`). To throw some more suggestions in the room: `(Active)SpanProvider`, `(Active)SpanRegistry`, `SpanCaptor`.